eslzArm/managementGroupTemplates/policyDefinitions/Databricks/DENY-Databricks-Sku.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Authorization/policyDefinitions",
+            "apiVersion": "2020-03-01",
+            "name": "Deny-Databricks-Sku",
+            "properties": {
+                "policyType": "Custom",
+                "mode": "Indexed",
+                "displayName": "Deny non-premium databricks sku",
+                "description": "Enfoces the use of Premium Databricks workspaces.",
```

Comment:
Pls expand on the description per our discussion, to highlight this is to ensure appropriate security features, RBAC etc.

Reply:
DONE